gcn/notices/km3net/test/medal_ranking_alert.example.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/test/medal_ranking_alert.schema.json",
+  "mission": "KM3NeT",
+  "instrument": "ARCA028",
+  "messenger": "Neutrino",
+  "packet_type": 999,
+  "alert_tense": "test",
+  "alert_type": "initial",
+  "alert_datetime": "2024-09-01T12:01:00.00Z",
+  "analysis_pipeline": "exceptionnal_evt_arca",
+  "description": "KM3NeT online analysis, bronze candidate neutrino observation.",
+  "event_name": ["KM240901A"],
+  "trigger_time": "2024-09-01T12:00:00.00Z",
+  "ra": 13.82,
+  "dec": 19.01,
+  "ra_dec_error": 0.9,
+  "healpix_url": "https://www.km3net.org/about-km3net/open-access/",
+  "far": 8.029e-8,
+  "additional_info": "Track only / Track+Shower analysis. Up-going / All-sky selection. Analysis pipeline event selection tuned to select X event per month in average.",
+  "triggering_evts": [
```

Comment:
How this ra,dec is different from the event above?

Reply:
In this case (where the alert is produced by only one event) there is no difference. 
But we have analysis pipeline where several events (coming from the same direction in a short time window, but individually below our threshold to send alert) would be identified as a signal of interest and lead to alert creation. 
In that case, the coordinates in "triggering_evts" are the coordinates of our individual events, while the "ra", "dec" and "ra_dec_error" of the alert body correspond to the most probable direction. 

We wanted to have an alert message as generic as possible, this is why we decided to duplicate the information, but I am open to suggestions if you have.

Reply:
Ok thanks.
Localization (ra,dec) is used at two places. 

(---)"ra": 13.82,
  "dec": 19.01,
  "ra_dec_error": 0.9,
  "healpix_url": "https://www.km3net.org/about-km3net/open-access/",
  "far": 8.029e-8,
  "additional_info": "Track only / Track+Shower analysis. Up-going / All-sky selection. Analysis pipeline event selection tuned to select X event per month in average.",
  "triggering_evts": [
{
      "trigger_time": "2024-09-01T12:00:00.00Z",
(---)      "ra": 13.82,
      "dec": 19.01,

I would suggest, either put all localizations in trigger_evts (ra, dec) list, Or create two examples files for one schema. One for alert, another for several events analysis pipeline.
It's creating confusion, example, whether above one is follow-up event.

Reply:
Indeed it could be confusing. I will create a second example file with several events in the alert. We are also discussing to rename "triggering_evts" into something more explicit ("hits_information" or "internal_triggers_information" or something else). I will commit the change as soon as we reach an internal agreement. 
The descriptions in the schema file will be updated as well (both for the alert coordinates and the triggering_evts variable).